packages/components/src/components/data-table/_data-table-inline-edit-cell.scss

```diff
@@ -315,8 +315,8 @@
     z-index: 10;
     width: rem(10px);
     height: rem(10px);
-    background-color: $inverse-01;
-    border: 1px solid $ui-03;
+    background-color: $background-inverse;
```

Comment:
Same comment: `$background-inverse` pairs with `$inverse-02` not `-01`. So this should either be `$icon-inverse` or `$text-inverse`